scripts/obtain_user_auth.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This program obtains a set of user credentials needed by the system tests
+for google.oauth2.credentials."""
+
+import json
+import os
+
+from oauth2client import client
+from oauth2client import tools
```

Comment:
Lol we're explicitly not implementing the authorization flow for 1.0.0. Once we actually add that feature, we'll remove oauth2client from this script.